version.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
```

Comment:
We're trying to shift away from using bash for build scripts & tooling, and using small go programs wherever possible.

If you can, please update it to use go. @zmb3 found [this library](https://github.com/bitfield/script) the other day, which may help, and [go-git](https://github.com/go-git/go-git) is useful for manipulating repositories.

Also: Beware that `go-git` is not a fully-featured `git` implementation and may not have what you need. If you go down this path and get hit by one of these omissions, my advice is not try & force `go-git`to handle it - just shell out to the command line `git`. I've lost _days_ fighting with go-git, and while it's very good when it works, when it doesn't work it just. doesn't. work.

(My 2c)